mapper/mapper_test.go

```diff
@@ -677,3 +697,28 @@ func TestCreatePreviousReleasesPage(t *testing.T) {
 		})
 	})
 }
+
+func TestMapLatestRelease(t *testing.T) {
+	t.Parallel()
+
+	Convey("When mapLatestRelease is called", t, func() {
+		Convey("with a date that should match one item in response", func() {
+			page := model.SearchPage{}
+			searchResponse, _ := GetMockSearchResponse()
+			mapResponse(&page, searchResponse, []data.Category{})
+			latestReleaseDate := "2015-02-17T00:00:00.000Z"
+
+			mapLatestRelease(&page, latestReleaseDate)
+			So(page.Data.Response.Items[0].IsLatestRelease, ShouldBeTrue)
+		})
+		Convey("with a date that doesn't match one item in response", func() {
+			page := model.SearchPage{}
+			searchResponse, _ := GetMockSearchResponse()
+			mapResponse(&page, searchResponse, []data.Category{})
+			latestReleaseDate := "2022-02-17T00:00:00.000Z"
+
+			mapLatestRelease(&page, latestReleaseDate)
+			So(page.Data.Response.Items[0].IsLatestRelease, ShouldBeFalse)
```

Comment:
You could use a Convey step here with a 'Then....' step here to make this more obvious what you're doing.

Reply:
👍